adhocracy4/comments/templatetags/react_comments.py

```diff
@@ -44,6 +49,7 @@ def react_comments(context, obj):
         'isModerator': is_moderator,
         'user_name': user_name,
         'isReadOnly': not has_comment_permission,
+        'would_have_perm': would_have_comment_permission,
```

Comment:
the default javascript style is to use camelCase variables. in meinBerlin we are using a linter which checks it. i'd prefer to use `wouldHavePerm` 

or we could simplify the code overall and just use `isReadOnly` and `isAuthenticated`
`isReadOnly` would be set to `not has_comment_permission and not would_have_comment_permission` and the check within react would be 
```
if (this.context.isAuthenticated && !this.props.isReadOnly) {
// show
} else if (!this.props.isReadOnly) {
// login
} else {
// no party
}
```

i think it would simplify the code as it keeps the generic sounding `would_have_perm` out of the react component. 

but those are just my personal opinions. i would not veto against the current solution. especially as "user_name" is already not camelcase

Reply:
Good idea, I will use isReadOnly!